taxi_service/settings.py

```diff
@@ -56,7 +56,7 @@
 TEMPLATES = [
     {
         "BACKEND": "django.template.backends.django.DjangoTemplates",
-        "DIRS": [],
+        "DIRS": [os.path.join(BASE_DIR, "templates")],
```

Comment:
Please replace to `[BASE_DIR / "templates"]`

Do not forget to remove os import